dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -734,7 +738,7 @@ HRESULT WINAPI CRegFolder::GetDefaultColumn(DWORD dwRes, ULONG *pSort, ULONG *pD
 
 HRESULT WINAPI CRegFolder::GetDefaultColumnState(UINT iColumn, DWORD *pcsFlags)
 {
-    if (iColumn >= REGISTRY_COL_COUNT)
+    if (iColumn >= REGFOLDERCOLUMNCOUNT)
```

Comment:
Why not keeping the original name? Other code (see e.g. the [CControlPanelFolder.cpp](https://github.com/reactos/reactos/pull/5743/files#diff-375e0fdced1bf197e4fd69a87008444b72d8b1a485148821c723094bba7d886a)) uses the `xxx_COL_COUNT` naming scheme.

Reply:
It uses a different name on purpose as to not confuse it with an actual column named COUNT.

Reply:
ah ok